File: Python3/salario.py
"""Crie um algoritmo para calcular o salário líquido de um funcionário,
   considerando que seu salário bruto, incide um desconto de 9% em inss 
   para a previdência. O algoritmo deve mostrar o nome do funcionário,
   o seu salário bruto, o valor de desconto de inss e o seu salário 
   líquido.(dica.: Você deverá pedir (ler) o nome do funcionário e 
   o valor do salário bruto)"""
 

class cadastro:
   nome = ""
   salario_bruto = 0.0

funcionario = cadastro()
funcionario.nome = input("Digite o nome do Funcionario: ")
funcionario.salario_bruto = int(input("Digite o valor do salario bruto "))
inss = (funcionario.salario_bruto * 9) / 100
salario_liquido = funcionario.salario_bruto - inss

print("O nome do funcionario = %s" %(funcionario.nome))
print("O salario bruto = %f" %(funcionario.salario_bruto))
print("O desconto do INSS = %f" %(inss))
print("O salario liquido  = %f" %(salario_liquido))
                  
   

File: java/salario.java
import java.util.Scanner;

class salario {
  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);
    float salario_liquido, salario_bruto, inss;
    String nome;
    System.out.println("Digite o nome do funcionario");
    nome = sc.nextLine();
    System.out.println("Digite o valor do salario bruto");
    salario_bruto = sc.nextFloat();
    inss = (salario_bruto * 9) / 100;
    salario_liquido = salario_bruto - inss;
    System.out.println("O nome do funcionario = " + nome + "\nO salario bruto = " + salario_bruto
        + "\nO desconto do inss = " + inss + "\nO salario liquido = " + salario_liquido);
    sc.close();
  }
}
